content/chapters/software-stack/lecture/slides.mk

```diff
@@ -0,0 +1,22 @@
+RVMD = reveal-md
```

Comment:
Why do you have this instead of using the generic makefile? Remove it from the PR and only use it locally.

Reply:
Will update it when PR #12 will be pushed. I wanted to give reviewers the possibility to built stuff without any headaches.